_includes/assets/js/indicatorModelHelpers.js

```diff
@@ -0,0 +1,539 @@
+(function() {
+  return {
+    UNIT_COLUMN: 'Units',
+    GEOCODE_COLUMN: 'GeoCode',
+    YEAR_COLUMN: 'Year',
+    VALUE_COLUMN: 'Value',
+    convertJsonFormat: function(data) {
+      var keys = Object.keys(data);
+      if (!keys.length) {
+        return [];
+      }
+
+      return data[keys[0]].map(function(item, index) {
+        return _.object(keys, keys.map(function(key) {
+          return data[key][index];
+        }));
+      });
+    },
+    extractUnique: function(prop, data) {
+      return _.chain(data).pluck(prop).uniq().filter(function(f) { return f; }).sortBy(function(year) {
```

Comment:
`extractUnique` isn't clear what it returns (unique what?). Also better to prefix the function with `get...` since it returns something directly.